.pipelines/azure-pipeline-build.yml

```diff
@@ -416,10 +417,14 @@ stages:
         displayName: "Build: Set values in payload.json for signing"
         condition: and(succeeded(), eq(variables.IS_MAIN_BRANCH, true))
 
-      - task: EsrpCodeSigning@3
-        displayName: "ESRP CodeSigning for Prometheus"
+      - task: EsrpCodeSigning@5
```

Comment:
revert the change branch name change (esrpUpdateSvcConnection) on line 5 and keep the display name of the tasks?

Do we have to sign the reference app image too or is the guidance just for images we serve to customers?